client/source/AppHolder.jsx

```diff
@@ -11,7 +11,7 @@ import ClickTracker from './utilities/clickTracker.js';
 // MAIN
 const AppHolder = function CreateAppHolder() {
   // STATES
-  const [prodID, setProdID] = useState(40344); // TODO: default view for 'no item searched yet'
+  const [prodID, setProdID] = useState(40444); // TODO: default view for 'no item searched yet'
```

Comment:
I believe the latest version of main has a completely different product ID in it than these two - I don't think the id variations impact my component but maybe check with others to see if the new number impacts their component render

Reply:
Changed back to 40355